src/main/resources/explicitIpPortRegistrator-example.xml

```diff
@@ -1,15 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <hazelcast id="hazelcast-consul-discovery" 
-           xsi:schemaLocation="http://www.hazelcast.com/schema/config hazelcast-config-3.6.xsd"
+           xsi:schemaLocation="http://www.hazelcast.com/schema/config hazelcast-config-4.0.xsd"
            xmlns="http://www.hazelcast.com/schema/config"
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
            
     <!-- NOTE! any XML value below can be variables that will be fetched from
          -D system properties @see http://docs.hazelcast.org/docs/3.6-EA/manual/html-single/index.html#using-variables  -->
-    <group>
```

Comment:
ditto above, why is this removed, used in example/travis no?

Reply:
The <group> tag has been removed in 4.0 and replaced by <cluster-name>

Reply:
The group tag has been removed in 4.0 and replaced by cluster-name